lib/shared/widgets/givt_secondary_elevated_button.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app_kids_web/shared/widgets/givt_elevated_button.dart';
+
+class GivtSecondaryElevatedButton extends StatelessWidget {
```

Comment:
Nice how you made components out of the different buttons :)

Reply:
thanks :)